src/components/Popup.js

```diff
@@ -1,21 +1,22 @@
 export default class Popup {
   constructor({ popupSelector }) {
     this._popupSelector = popupSelector;
-    this._popupElement = document.querySelector(popupSelector);
+    this.popupElement = document.querySelector(popupSelector);
+    this._form = this.popupElement.querySelector("form");
```

Comment:
`this._form = this.popupElement.querySelector("form");` is not needed in `Popup` because the image modal doesn't have any form